Tensile/KernelWriterAssembly.py

```diff
@@ -2223,6 +2225,16 @@ def defineBufferMemoryMacros(self):
       replace = f'{type_list[t]}' if (self.version[0] < 11) else f'{t}'
       kStr += self.generalMacro('buffer_load_', origin, replace, 'dst', 'voffset', 'base', 'soffset', 'offen', 'ioffset', 'md0', 'md1', 'md2') + self.endLine
 
+    # Extra macro for DirectToLds loads with no destination register
+    type_list = {
+      'b32'       : 'dword',
+      'u16'       : 'ushort'
```

Comment:
Do we support u16?

Reply:
Half+DTL doesn't work right now, but added for completeness since it was the only other data type in the list of macros that we generate that can also handle lds modifier